app/controllers/teams_controller.rb

```diff
@@ -1,5 +1,5 @@
 class TeamsController < ApplicationController
-  before_action :authenticate_user!, only: %i[add_member remove_member]
+  before_action :authenticate_user!, only: %i[add_member remove_member new create]
```

Comment:
You don't need to use `only: %i[add_member remove_member new create]` here since you are executing it for each action in this controller. `before_action :authenticate_user!` is enough

Reply:
But I have two more actions in this controller: index and show. So maybe it would be better to write it like this: ```
efore_action :authenticate_user!, except: %i[index, show]
```

Reply:
👍